martin-mbtiles/src/tile_copier.rs

```diff
@@ -632,11 +607,13 @@ mod tests {
         let dst =
             PathBuf::from("file:copy_to_existing_ignore_mode_mem_db?mode=memory&cache=shared");
 
+        // TODO: need a detailed comment explaining that due to `let _var = copy`,
+        //   connection remains open and the second copy command reuses the same in-memory file.
         let _dst_conn = copy_mbtiles_file(TileCopierOptions::new(dst_file.clone(), dst.clone()))
             .await
             .unwrap();
 
-        let mut dst_conn = copy_mbtiles_file(
+        let mut conn = copy_mbtiles_file(
```

Comment:
The different files we are using here get confusing; I would like to keep the name `dst_conn` to have a clear distinction and the reader does not have to think about it.